fr/integrations/plugin-packs/procedures/operatingsystems-aix-ssh.md

```diff
@@ -0,0 +1,169 @@
+---
+id: operatingsystems-aix-ssh
+title: AIX SSH
+---
+
+## Contenu du Plugin-Pack
+
+### Objets supervisés
+
+Le Plugin-Pack inclue la supervision le système AIX en utilisant les commandes SSH, tel que :
+* Retour de commande
+* Messages errpt
+* Inodes
+* Volumes Groupes
+* Processus
+* Stockage 
+
+### Métriques collectées
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--Cmdreturn-->
+
+| Metric name                               | Description                            | Unit |
+| :---------------------------------------- | :------------------------------------- | :--- |
+| command.exit.code.count                   | Number of exit code return             | count|
+
+<!--Inodes-->
+
+| Metric name                               | Description                            | Unit |
+| :---------------------------------------- | :------------------------------------- | :--- |
+| storage.inodes.usage.percentage           | Inodes usage in percentage	           | %    |
+
+<!--Process-->
+
+| Metric name                               | Description                            | Unit |
+| :---------------------------------------- | :------------------------------------- | :--- |
+| processes.alerts.count                    | Number of alerts processes             | count|
+| processes.total.count                     | Total number of alerts processes       | count|
+
+<!--Storage-->
+
+| Metric name                               | Description                            | Unit |
+| :---------------------------------------- | :------------------------------------- | :--- |
+| storage.space.usage.bytes                 | Storage space usage                    | B    |
+| storage.space.free.bytes                  | Storage free space usage               | B    |
+| storageresource.space.usage.percentage    | Storage percentage space usage         | %    |
+
+
+<!--END_DOCUSAURUS_CODE_TABS-->
+
+## Prérequis
+
+Afin de fonctionner, le Plugin nécessite une connexion SSH entre le Poller et le serveur executant AIX SSH. L'utilisateur distant
```

Comment:
Provide a test or an example of the command to make sure people understand that the connection is made from centreon-engine user.

Reply:
PS: do it in the EN version also please :)